src/_templates/react-native-typescript-screen/new/component.ejs

```diff
@@ -0,0 +1,56 @@
+---
+to: <%= root %>/<%= path %>/<%= h.inflection.transform(name, ['undasherize', 'classify']) %>/<%= h.inflection.transform(name, ['undasherize', 'classify']) %>.tsx
+---
+<% const comp = h.inflection.transform(name, ['undasherize', 'classify']) -%>
+<% if(componentType === 'Class') { -%>
```

Comment:
Do we need to keep class components?

Reply:
![image](https://user-images.githubusercontent.com/3806031/68618490-a9da3480-0486-11ea-9f8b-a5a42f7c5485.png)

Reply:
will discuss as a group later.